src/_includes/base.liquid

```diff
@@ -44,15 +44,15 @@ menu: []
     <meta name="description" content="{{ description }}" />
     <meta name="viewport" content="width=device-width" />
     <meta property="og:title" content="{{ full_title }}" />
-    <link rel="stylesheet" href="/static/styles.css" />
+    <link rel="stylesheet" href="css/styles.css" />
```

Comment:
Since this is a template we might use in a document which lives in a directory, we should use an absolute path.

```suggestion
    <link rel="stylesheet" href="/css/styles.css" />
```